integration/integration_test.go

```diff
@@ -365,7 +365,8 @@ func TestBuildFromDockerfile(t *testing.T) {
 	require.Equal(t, "hello", strings.TrimSpace(output))
 
 	// Verify that the Docker configuration secret file is removed
-	output = execContainer(t, ctr, "stat "+filepath.Join(constants.MagicDir, "config.json"))
+	configJSONContainerPath := filepath.Join(constants.MagicDir("").String(), "config.json")
```

Comment:
Idea: instead of `constants.MagicDir("")` why not add a func like `constants.DefaultMagicDir()`?